MaxText/layers.py

```diff
@@ -39,8 +39,14 @@
 from jax.ad_checkpoint import checkpoint_name
 import jax.numpy as jnp
 from jax.experimental.pallas.ops.tpu import flash_attention
+import os
+import math
 
-
+try:
+  from jax.experimental.pallas.ops import attention
```

Comment:
I think we should be more careful when namespacing here, say calling these `as pallas_attention` and `as layer_norm`?